### ISSUE TITLE
user view and personal area

### ISSUE THREAD

**Issue by PietroLiuzzo** [BetaMasaheft/Documentation#639]
A user should be able to 
1. [APP10](APP10) - create her/his own account and manage it, changing password for example.
2. [APP11](APP11) - Save results to personal account with a log like process which will save a query to a list of queries for each user. see log.xqm code.
3. [APP12](APP11) - Retrive results from personal account
4. See its commits and on what he has worked (simple links for git commits, and further reporting from the app)
5. import also assigned issues from git.

**Comment by PietroLiuzzo** [comment]
the new module log.xqm contains at the moment two functions which add log entries in the collection log file bm-log.xml. Each of these log records registers the users, the type of the log and the url or relevant path.
- users are always obtained with xmldb:get-current-user()
- types are: 
          - **page** for all pages using templates.xql, and actually use the function stored in app.xqm app:logging() 
          - **error** for requests landing on the error.xqm module
          - **query** for queries regardless if they come from simple or advanced search
          - **Xpath** for the xpath function built by the app:query() function
          - **Xpath query** for the xpath function built by the app:xquerypath() function
          - the restxq module name (REST, iiif, item, list, viewer, dts) - no error logging as yet been added.

**Comment by PietroLiuzzo** [comment]
- a new module user.xqm contains a rest path to the user page
- controller.xql updated to redirect to restxq also for these requests

**Comment by PietroLiuzzo** [comment]
This is now implemented and the user can see the information liste above and perform those actions but works only after login on the server. logout does not work.

**Comment by PietroLiuzzo** [comment]
I have a small form in the nav bar that allows the user to enter un and pw, and the controller creates a session using the xmldb and login module. this works well until I try to make a request which the controller forwards to restxq. At that point the session is lost. 

Same happens if I login in a restxq request with $rest:POST and then reload that with a further parameter for example. login data is lost. 

I also tried to set and later get parameters and attributes on the request, without success (checking with (session:get-attribute-names()) and fairly enough I found no way I can find out the password from the username in the securitymanager module.
